packages/validator-ajv8/test/validator.test.ts

```diff
@@ -12,11 +12,19 @@ import {
 
 import AJV8Validator from "../src/validator";
 import { Localizer } from "../src";
+import { type ErrorObject } from "ajv";
```

Comment:
Move this to the top of the file please, and I don't believe you need the `type` either

Reply:
I'll commit this but I think the type is important because that imports only the types and none of the bundled javascript

Reply:
In the scope of the test it is probably not that big of a deal